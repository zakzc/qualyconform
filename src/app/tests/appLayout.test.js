import React from "react";
import Enzyme, { shallow } from "enzyme";

import Adapter from "@wojtekmaj/enzyme-adapter-react-17";

import AppLayout from "../../App";

Enzyme.configure({ adapter: new Adapter() });

describe("Main layout component", () => {
  const appLayout = shallow(<AppLayout />);
  describe("Initiates correctly", () => {
    it("renders the app", () => {
      expect(appLayout).toMatchSnapshot();
    });
    it("has the proper navigation import", () => {
      expect(appLayout.find("Navigation")).toBeTruthy();
    });
    it("has the proper routes import", () => {
      expect(appLayout.find("Routes")).toBeTruthy();
    });
  });
});
