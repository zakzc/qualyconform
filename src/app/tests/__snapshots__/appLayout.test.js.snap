// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main layout component Initiates correctly renders the app 1`] = `ShallowWrapper {}`;
